wasm_builder_cli/README.md

```diff
@@ -0,0 +1,23 @@
+# `iroha_wasm_builder_cli`
+
+A CLI around [`iroha_wasm_builder`](../wasm_builder) crate.
+
+## Usage
+
+**Check the smartcontract:**
+
+```bash
+iroha_wasm_builder_cli check path/to/project
+```
+
+**Build the smartcontract:**
+
+```bash
+iroha_wasm_builder_cli build path/to/project
+```
+
+**Build with options:**
+
+```bash
+iroha_wasm_builder_cli build path/to/project --optimize --format
+```
```

Comment:
Maybe document how the `default_validator` should be built? It's a quite a common thing to want (for a developer).. AFAIU it's `iroha_wasm_builder_cli build default_validator --optimize > ./configs/peer/validator.wasm`

Reply:
Documented in `default_validator/README.md`. I am not sure this is a good location for it, but I haven't found any current documentation to patch.